tyk_reverse_proxy_clone.go

```diff
@@ -238,7 +236,7 @@ func TykNewSingleHostReverseProxy(target *url.URL, spec *APISpec) *ReverseProxy
 
 		// No override, and no load balancing? Use the existing target
 		targetToUse := target
-		if switchTargets {
```

Comment:
Agreed, will undo. This does change functionality somewhat, as empty strings will be treated differently.